File: Difficulty: Easy/Average/average.java
class Solution {
    public static double posAverage(int[] arr) {
        // Code here
        double sum=0,avg;
        int count=0;
        for(int i=0;i<arr.length;i++){
            if(arr[i]>=0){
                sum=sum+arr[i];
                count+=1;
            }
        }
        avg=sum/count;
        return avg;
    }
}

File: Difficulty: Easy/Average/average.cpp
int posAverage(int arr[], int size) {
    int sum = 0, num = 0;

    for(int i = 0; i < size; i++){
        if(arr[i] >= 0){   // include 0
            sum += arr[i];
            num++;
        }
    }

    if(num == 0) return 0;

    return sum / num;
}